docs/quickstart.md

```diff
@@ -79,8 +80,24 @@ postgres-example-008   Waiting     cpu=1198, memory=2701
 
 The trials will run until the `experimentBudget` is satisfied. In this example, there will be 40 trials that run.
 While the trial is running, there may be assignment combinations that are unstable and result in a failure.
-
 After the trial is complete, you will be able to view the parameters and the metrics generated from the trial.
+
+## Add the Current Configuration as a Baseline
+
+Optionally, you can compare Red Sky Ops results to a baseline by running a trial with the parameters that are currently set in the the application manifests.
+While your experiment is still running, use the following command to suggest a trial.
+
+```sh
+redskyctl suggest postgres-example --interactive --labels 'baseline=true'
```

Comment:
Only nit, can you begin this line with `$ `

Reply:
ah, yes thank you!